gaia-fast-list.js

```diff
@@ -1058,9 +1031,12 @@ Internal.prototype = {
     var bottom = this.el.bottom;
     var top = this.el.top;
 
-    return (top != null && bottom != null)
-      ? parent.innerHeight - top - bottom
-      : this.el.clientHeight;
+    if (top != null && bottom != null) {
+      return parent.innerHeight - top - bottom;
+    }
+
+    return parseInt(this.el.style.height)
```

Comment:
This avoids a reflow on startup if the component has `style.height`. Sorta unrelated but ... meh.